routers/actionsRouter.js

```diff
@@ -0,0 +1,140 @@
+// library
+const express = require('express');
+
+// database file
+const projects = require('../data/helpers/projectModel');
+const actions = require('../data/helpers/actionModel');
+
+// router handler
+const router = express.Router({
+  mergeParams: true
```

Comment:
can you explain what `mergeParams` does?

Reply:
I did forget about this. mergeParams allows us to pull in the req.params values from the parent router (projectsRouter) so that the value in child router (actionsRouter) takes precedence.